page-container/index-page/IndexPageContainer.tsx

```diff
@@ -0,0 +1,108 @@
+import { Box, Grid, makeStyles } from '@material-ui/core';
+import { Header, Footer } from 'pedialab-shared/components';
+import {
+  HeroContent,
+  OurCompany,
+  SoftwareDelivery,
+  VendorDifference,
+  TechleadaaS,
+  ProjectScopes,
+  TeamUpWithTechlead,
+  DeveloperSupport,
+  OurTeam,
+  CompaniesWeWorkWith,
+  WhereWeServe,
+  CaseStudies,
+  ProjectPhrases,
+  Testimonial,
+  Contact
+} from 'pedialab-pages/index-page';
+
+const useStyle = makeStyles({
+  heroContent: {
+    marginTop: '7rem'
+  },
+  ourCompany: {
```

Comment:
I see you're using mx={} or mt={} at different places already, do you think we should align to use those syntax?

Reply:
The syntax `mx`, `mt` can only used with `<Box>` Material-UI element, and each one will create a new class.
Which is bad in loop for child element, will create too many redundant class and CSS code.

However, I'm not noticed this when the beginning. After that, I prefer using `<Grid>` for flex layout, and other styling with `makeStyles()`, this can be replaced by styled-component way.